packages/core/ui/App.js

```diff
@@ -89,8 +90,18 @@ const App = observer(({ session, HeaderButtons }) => {
     name,
     menus,
     views,
+    configuration,
   } = session
 
+  function renderLogo() {
```

Comment:
Just a style note: this concept of a "function that produces JSX" should really be it's own react component

Reply:
Ahh, got it